Ports/libfts/package.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='libfts'
+version='1.2.7'
+files="https://github.com/void-linux/musl-fts/archive/refs/tags/v${version}.tar.gz v${version}.tar.gz 49ae567a96dbab22823d045ffebe0d6b14b9b799925e9ca9274d47d26ff482a6"
+workdir="musl-fts-${version}"
+auth_type='sha256'
+useconfigure='true'
+
+pre_configure() {
+    # FIXME: The maintainer forgot to run autoconf before creating the tarball.
+    # Remove this when the next update is released.
```

Comment:
This archive will never have a preconfigured `configure` file.

Reply:
Makes sense, tbh these lines were copied from the jq port where the comment makes sense.